cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -5,10 +5,95 @@ include::modules/common-attributes.adoc[]
+
+////
+
+[id="odo-getting-support_{context}"]
+== Getting support
+
+.For Documentation
+
+If you have found an error or have suggestions for improving the documentation, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *{product-title}* product type and the *Documentation* component type.
+
+.For Product
+
+If you have found an error, encountered a bug or have suggestions for improving the functionality of {odo-title}, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *Red Hat odo for OpenShift Container Platform* product type.
+
+Provide as many details in the description of the issue as possible.
+
+[id="odo-fixed-known-issues_{context}"]
+== Fixed known issues
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760573[Bug 1760573] An active pointer is not pointing to a current active project.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760578[Bug 1760578] `odo watch` does not fail with error message for git based components but it should.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760583[Bug 1760583] `odo config unset` does not unset an environment variable but says it did.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760585[Bug 1760585] `odo delete --all` deletes the`$HOME/.odo` folder when run from `$HOME`.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760589[Bug 1760589] Autocompletion does not work for  `--context` in `odo push`.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1761442[Bug 1761442] `component create` fails with flag `--context` and `--binary` if the binary is located in a temporary folder.
+
+[id="odo-known-issues_{context}"]
+== Known issues
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760574[Bug 1760574] A deleted namespace is being listed in the `odo project get` command.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760575[Bug 1760575] `odo app delete` removes application components but not services.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760577[Bug 1760577] `odo push` does not delete OpenShift objects when the component name is changed.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760586[Bug 1760586] `odo delete` starts an infinite loop after project is deleted and a component name is set.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760588[Bug 1760588] `odo service create` crashes when run in Cygwin.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760590[Bug 1760590] In Git BASH for Windows `odo login -u developer` does not hide a typed password when requested.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1783179[Bug 1783179] If env var is set after first component push then route URL doesn't work after editing the source
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1783188[Bug 1783188] In a disconnected cluster component create throws error of `...tag not found...` though it is listed in catalog list
```

Comment:
comma after cluster
the `component create` command?
an error
throws/displays
the catalog